### PR TITLE
Create DIS_2.45.md

### PR DESCRIPTION
**Pull request by PedroDebevere**

### CODE REVIEW

File: release-notes/DIS/DIS_2.45.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+The SLDisCsharpAnalysis project has been extracted from DIS and new NuGet packages have been created/extend to hold this extracted functionality:
+
+New packages:
+
+- Skyline.DataMiner.CICD.CSharpAnalysis.Common
+
+- Skyline.DataMiner.CICD.CSharpAnalysis.Protocol
+- Skyline.DataMiner.CICD.Models.Common
+
```
Comment (MichielOda) [inline release-notes/DIS/DIS_2.45.md:56]:
```suggestion
- Skyline.DataMiner.CICD.CSharpAnalysis.Common
- Skyline.DataMiner.CICD.CSharpAnalysis.Protocol

- Skyline.DataMiner.CICD.Models.Common

```

File: release-notes/DIS/DIS_2.45.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+Removed old validator error 2603: RawType double has no length definition.
+
+## Changes
+
+### Enhancements
+
+#### Driver help generator now generates markdown [ID_37313]
+
+The driver help generation in DIS has been updated so that it now generates a markdown file (.md) to be used in the docs repository instead of an HTML page that was used on DCP. The menu item that allowed to open the driver help on DCP has been renamed and will now open the corresponding page on the GitHub repository (if it exists).
```
Comment (MichielOda) [inline release-notes/DIS/DIS_2.45.md:42]: Menu item isn't actually renamed.

File: release-notes/DIS/DIS_2.45.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+- Skyline.DataMiner.CICD.CSharpAnalysis.Common
+
+- Skyline.DataMiner.CICD.CSharpAnalysis.Protocol
+- Skyline.DataMiner.CICD.Models.Common
+
+Extended packages:
+
+Skyline.DataMiner.CICD.Models.Protocol
+
+TThe SLDisCompiler project has been extracted from DIS and the functionality from this project has been migrated to the already existing NuGet package "Skyline.DataMiner.CICD.Parsers.Protocol" and the newly introduced NuGet package "Skyline.DataMiner.CICD.Assemblers.Protocol".
```
Comment (MichielOda) [inline release-notes/DIS/DIS_2.45.md:61]:
```suggestion
The SLDisCompiler project has been extracted from DIS and the functionality from this project has been migrated to the already existing NuGet package "Skyline.DataMiner.CICD.Parsers.Protocol" and the newly introduced NuGet package "Skyline.DataMiner.CICD.Assemblers.Protocol".
```

**Review by MichielOda — CHANGES_REQUESTED** [review]

File: release-notes/DIS/DIS_2.45.md (hunk 2 above)
Comment (PedroDebevere) [reply release-notes/DIS/DIS_2.45.md]: updated

**Review by MichielOda — APPROVED** [review]

**Review by spetebroodt — APPROVED** [review]

**Comment by spetebroodt** [comment]
@MichielOda @PedroDebevere Can you double-check my corrections before I merge this?

**Comment by MichielOda** [comment]
> @MichielOda @PedroDebevere Can you double-check my corrections before I merge this?

Looks good to me. Once Pedro confirms, then this can be merged.

**Comment by PedroDebevere** [comment]
updates are ok

**Pull request merged**